packages/react-storage/src/components/StorageImage/StorageImage.tsx

```diff
@@ -23,7 +23,10 @@ export const StorageImage = ({
 
   const url = useStorageURL({
     key: imgKey,
-    options,
+    options: {
+      ...options,
+      validateObjectExistence: true,
```

Comment:
Don't feel strongly but should `validateObjectExistence` be a prop with a default value of `true`?

Reply:
Done